src/Files.App/ViewModels/SidebarViewModel.cs

```diff
@@ -236,6 +236,7 @@ public SidebarViewModel()
 			SideBarItems = new BulkConcurrentObservableCollection<INavigationControlItem>();
 			EmptyRecycleBinCommand = new RelayCommand<RoutedEventArgs>(EmptyRecycleBin);
 			UserSettingsService.OnSettingChangedEvent += UserSettingsService_OnSettingChangedEvent;
+			RecycleBinHelpers.RecycleBinChanged += OnRecycleBinChanged;
```

Comment:
Unsure, but the event should probably be detached to avoid memory leaks (unless we don't have to do it)